docs/ROADMAP.md

```diff
@@ -1,19 +1,19 @@
 # Roadmap
 
-## Baseline
+## Baseline - What We Have Done
 
 <table>
   <tr><td> x </td><td> <img src="/docs/images/noun_milestone_2525794.svg"> </td><td> Milestones documented, project description written, and initial OPAT schema and catalog drafts published. </td></tr>
-  <tr><td> x </td><td> <img src="/docs/images/noun_yaml_file_document_2760448.svg"> </td><td> Create the highly structured, machine-readable document format. YAML was chosen as the markup language. Refine the YAML OPAT schema to support the requirements of the current Section 508 version of VPAT. </td></tr>
-  <tr><td> x </td><td> <img src="/docs/images/noun_construct_validity_3159900.svg"> </td><td> Ensure that YAML documents conform to a defined structure. Build in validation code for the OPAT schema so that key elements are evaluated. </td></tr>
-  <tr><td> x </td><td> <img src="/docs/images/noun_continuous_deployment_2661276.svg"> </td><td> Build testing into the development so that errors are caught early in the process. Continuous integration should be used to validate the code and process. </td></tr>
+  <tr><td> x </td><td> <img src="/docs/images/noun_yaml_file_document_2760448.svg"> </td><td> Create the <a href="/catalog/2.4-edition-wcag-2.0-508-en.yaml">highly structured, machine-readable document format</a>. YAML was chosen as the markup language. Refine the YAML OPAT schema to support the requirements of the current Section 508 version of VPAT. </td></tr>
+  <tr><td> x </td><td> <img src="/docs/images/noun_construct_validity_3159900.svg"> </td><td> Ensure that <a href="/schema/opat-0.1.0.json">YAML documents conform</a> to a defined structure. Build in validation code for the OPAT schema so that key elements are evaluated. </td></tr>
```

Comment:
Can you also add the link to the catalog schema?

Reply:
I can add /schema/opat-catalog-0.1.0.json but then I need to change the text. 

Would this work:

`Ensure that YAML documents conform to a defined <a href="/schema/opat-0.1.0.json">structure</a> and <a href="/schema/opat-catalog-0.1.0.json">catalog</a>. Build in validation code for the OPAT schema so that key elements are evaluated.`